### PR TITLE
Give the Paintslinger Lens the ability to paint all dyeable blocks

### PR DESCRIPTION
**Pull request by ToMe25**
Add Concrete, Concrete Powder and Glazed Terracotta blocks to the list of blocks that can be dyed with the Paintslinger lens.
Add unstained glass, glass panes and terracotta to the blocks the Paintslinger lens can paint.

Edit: I don't think adding Shulker Boxes would be a good idea.

### CODE REVIEW

File: src/main/java/vazkii/botania/common/core/handler/IMCSender.java [hunk 1]
```diff
@@ -87,24 +87,30 @@ public static void enqueue(InterModEnqueueEvent evt) {
...
-		Block[] stainedGlass = { Blocks.WHITE_STAINED_GLASS, Blocks.ORANGE_STAINED_GLASS, Blocks.MAGENTA_STAINED_GLASS, Blocks.LIGHT_BLUE_STAINED_GLASS,
+		Block[] glass = { Blocks.GLASS, Blocks.WHITE_STAINED_GLASS, Blocks.ORANGE_STAINED_GLASS, Blocks.MAGENTA_STAINED_GLASS, Blocks.LIGHT_BLUE_STAINED_GLASS,
 				Blocks.YELLOW_STAINED_GLASS, Blocks.LIME_STAINED_GLASS, Blocks.PINK_STAINED_GLASS, Blocks.GRAY_STAINED_GLASS, Blocks.LIGHT_GRAY_STAINED_GLASS,
 				Blocks.CYAN_STAINED_GLASS, Blocks.PURPLE_STAINED_GLASS, Blocks.BLUE_STAINED_GLASS, Blocks.BROWN_STAINED_GLASS, Blocks.GREEN_STAINED_GLASS, Blocks.RED_STAINED_GLASS, Blocks.BLACK_STAINED_GLASS };
-		for(Block b : stainedGlass)
+		for(Block b : glass)
 			send(IMC.REGISTER_PAINTABLE_BLOCK, new PaintableBlockMessage(ColorHelper.STAINED_GLASS_MAP::get, b));
 
-		Block[] sgPanes = { Blocks.WHITE_STAINED_GLASS_PANE, Blocks.ORANGE_STAINED_GLASS_PANE, Blocks.MAGENTA_STAINED_GLASS_PANE, Blocks.LIGHT_BLUE_STAINED_GLASS_PANE,
+		Block[] glass_panes = { Blocks.GLASS_PANE, Blocks.WHITE_STAINED_GLASS_PANE, Blocks.ORANGE_STAINED_GLASS_PANE, Blocks.MAGENTA_STAINED_GLASS_PANE, Blocks.LIGHT_BLUE_STAINED_GLASS_PANE,
```
Comment (williewillus) [inline src/main/java/vazkii/botania/common/core/handler/IMCSender.java]: camelCase please

Comment (ToMe25) [reply src/main/java/vazkii/botania/common/core/handler/IMCSender.java]: OK sec ill change that quick

Comment (ToMe25) [reply src/main/java/vazkii/botania/common/core/handler/IMCSender.java]: should be done.

**Comment by ToMe25** [comment]
thanks

**Pull request merged**